dev/com.ibm.ws.transport.http/src/com/ibm/wsspi/http/ee7/HttpInputStreamEE7Extended.java

```diff
@@ -96,9 +96,11 @@ public boolean isFinished() {
 
     private boolean setNextBuffer() {
         if (this.postDataPos < this.postData.length) {
-            this.buffer = HttpDispatcher.getBufferManager().allocate(this.postData[this.postDataPos].length);
-            this.buffer.put(this.postData[this.postDataPos]);
-            this.buffer.flip();
+            this.buffer = HttpDispatcher.getBufferManager().wrap(this.postData[this.postDataPos]);
+            this.postData = null;
```

Comment:
It was not my intent that we should null out postData.   That was a transcription problem from my original thought.  It would lead to a NPE later on.  My intent originally was to do `this.postData[this.postDataPos] = null;`, but I was concerned that that would be bad if the postData array was used later on again with additional processing of the request.  If that is not a concern that it will be used later and prematurely nulling it out isn't an issue, then we could stay with that.  Regardless this line should change to either be removed or only null out the specific array entry and not the entire array.